packages/contracts/contracts/libraries/Types.sol

```diff
@@ -189,4 +189,25 @@ library Types {
         uint256 value;
         bytes data;
     }
+
+    /**
+     * @notice Struct representing the data for verifying the public input.
+     *
+     * @custom:field srcOutputRootProof          Proof of the source output root.
+     * @custom:field dstOutputRootProof          Proof of the destination output root.
+     * @custom:field publicInput                 Ingredients to compute the public input used by ZK proof verification.
+     * @custom:field rlps                        Pre-encoded RLPs to compute the next block hash of the source output root proof.
```

Comment:
Is exceeding the max length allowed?

Reply:
Is there a maximum length in solidity formatting rule?

Reply:
Yes, it is.
FYI, https://docs.soliditylang.org/en/latest/style-guide.html#maximum-line-length

Reply:
I wrapped line.